### PR TITLE
Add test for cli usage with directory input

### PR DESCRIPTION
**Pull request by autra**
Here is a little test for directory usage of `cli.js`. WDYT @ahdinosaur? Especially, I'm not sure about the creation of the `dummy` directory, maybe I should use `tmp` instead?

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -56,3 +59,35 @@ test('simple transpiler using browserify transforms', function (t) {
...
+      if (error) {
+        t.fail(error)
+        return
+      }
+
+      // assert stuff
+      vinylfs.src(['dummy/**/*'])
+      .pipe(through2.obj(function (file, enc, next) {
+        var actualFile = fs.readFileSync(file.path)
+        var expectedFileName = './test/fixtures/child-expected/' + file.stem + file.extname
```
Comment (ahdinosaur) [inline test/index.js]: these are missing yeah?

**Review by ahdinosaur — APPROVED** [review]
hey @autra this looks great!

yeah, using `dummy` works, but using `tmp` makes sense.

File: test/index.js (hunk 1 above)
Comment (autra) [reply test/index.js]: :-O

**Comment by autra** [comment]
@ahdinosaur I've just added the missing files. Btw, the `simple transpiler using browserify transforms` test is failing on my machine (seems unrelated to my PR).

**Comment by ahdinosaur** [comment]
> Btw, the simple transpiler using browserify transforms test is failing on my machine (seems unrelated to my PR).

yeah, fails on my machine as well, something looks to have changed in the source map output. feel free to make a pull request fixing this.

**Pull request merged**